### DISCUSSION TITLE
Recommendations for video transcoding settings?

### DISCUSSION THREAD

**Discussion by Qhilm** [immich-app/immich#11272] [General]
What settings do you use?

My server users have firefox mostly, and the mobile app for iOS.

What would be the recommended setting for this particular clients?

And is there a settings that has a good ratio between bandwidth efficiency and compatibility? (I don't care so much for CPU efficiency at the moment)

**Comment by krypton9208** [comment]
I'm thinking about buy some GPU, to help transcoding, but, i don't find any list of with graphic cards immich handled.
Can someone give me examples with compatible cards?
